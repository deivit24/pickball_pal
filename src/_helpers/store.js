import { createStore, applyMiddleware } from 'redux';
import { composeWithDevTools } from 'redux-devtools-extension';
import thunk from 'redux-thunk';
import rootReducer from '../_reducers/root';

//Creating the store
const store = createStore(
  rootReducer,
  composeWithDevTools(applyMiddleware(thunk))
);

export default store;
