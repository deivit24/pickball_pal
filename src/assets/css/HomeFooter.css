#HomeFooter {
  height: 300px;
  background: #3f50b5;
  color: white;
  display: flex;
  flex-direction: column;
  justify-content: center;
  text-align: center;
}
