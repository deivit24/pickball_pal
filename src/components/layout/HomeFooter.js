import React from 'react';
import '../../assets/css/HomeFooter.css';

const HomeFooter = () => {
  return (
    <div id="HomeFooter">
      <div className="container">
        <h1>
          Created by{' '}
          <a href="https://deivitsalazar.com" target="a_blank">
            {' '}
            David Salazar
          </a>
        </h1>
      </div>
    </div>
  );
};

export default HomeFooter;
