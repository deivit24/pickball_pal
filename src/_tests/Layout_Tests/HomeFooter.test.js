import React from 'react';
import { render } from '@testing-library/react';
import HomeFooter from '../../components/layout/HomeFooter';
import { MemoryRouter } from 'react-router';
import { Provider } from 'react-redux';
import store from '../../_helpers/store';

//Testing HomeFooter

describe('Testing the HomeFooter', () => {
  it('it renders HomeFooter without crashing', () => {
    render(
      <MemoryRouter>
        <Provider store={store}>
          <HomeFooter />
        </Provider>
      </MemoryRouter>
    );
  });

  it('matches snapshot', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <Provider store={store}>
          <HomeFooter />
        </Provider>
      </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
