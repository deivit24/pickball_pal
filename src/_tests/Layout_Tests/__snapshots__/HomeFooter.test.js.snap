// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the HomeFooter matches snapshot 1`] = `
<DocumentFragment>
  <div
    id="HomeFooter"
  >
    <div
      class="container"
    >
      <h1>
        Created by 
        <a
          href="https://deivitsalazar.com"
          target="a_blank"
        >
           David Salazar
        </a>
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
